test/SlowTests/Server/Documents/DataArchival/DataArchivalDataSubscriptionsTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Utils;
+using Raven.Client;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations.DataArchival;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.Util;
+using SlowTests.Core.Utils.Entities;
+using Sparrow;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Server.Documents.DataArchival;
+
+
+    public class DataArchivalDataSubscriptionsTests: RavenTestBase
+    {
+        public DataArchivalDataSubscriptionsTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private async Task SetupDataArchival(IDocumentStore store)
+        {
+            var config = new DataArchivalConfiguration {Disabled = false, ArchiveFrequencyInSec = 100};
+
+            await DataArchivalHelper.SetupDataArchival(store, Server.ServerStore, config);
+        }
+        
+        
+    
+        [Fact]
+        public async Task ArchivedDocumentsDataSubscription()
```

Comment:
Can we have more meaningful name for this test? Something like `DataSubscriptionWillOperateOnOnlyOnArchivedDocuments`?

Reply:
94bf30efee5c41abff31d63dda4af0815ad53314 fixed here